### PR TITLE
Milestone One: Hello World Project

### PR DESCRIPTION
**Pull request by canon20paul**
Milestone One: Hello World Project

### CODE REVIEW

File: README.md [hunk 1]
```diff
@@ -1,2 +1,24 @@
...
+
+Authors
+👤 Paul Malongo
+
+GitHub: @canon20paul
+Twitter: @paulmalongo
+LinkedIn: Paulmalongo
+
+ License
+This project is MIT licensed.
```
Comment (muneebulrehman) [inline README.md]: Your readme file does not match the requirements of the project. Refer to [this](https://github.com/microverseinc/readme-template/blob/master/README.md) readme file on how to customize your readme file so it looks professional.

**Review by muneebulrehman — CHANGES_REQUESTED** [review]
Hi @canon20paul , 

Good job so far! I see you put a lot of effort into this project. However, there are some issues that you still need to work on to go to the next project but you are almost there!

## Highlights


## Required Changes ♻️
_Check the comments under the review._

## Optional suggestions
_Every comment with the [OPTIONAL] prefix is not crucial enough to stop the approval of this PR. However, I strongly recommend you to take them into account as they can make your code better._

Cheers and Happy coding!👏👏👏

Feel free to leave any questions or comments in the PR thread if something is not 100% clear.
Please, remember to tag me in your question so I can receive the notification.

**_Please, do not open a new Pull Request for re-reviews. You should use the same Pull Request submitted for the first review, either valid or invalid unless it is requested otherwise._**

------
_As described in the [Code reviews limits policy](https://microverse.zendesk.com/hc/en-us/articles/1500004088561) you have a limited number of reviews per project (check the exact number in your Dashboard). If you think that the code review was not fair, you can request a second opinion using [this form](https://airtable.com/shrQAqnBwek5a0O0s)._

File: README.md [hunk 2]
```diff
@@ -1,2 +1,41 @@
...
+Give a ⭐️ if you like this project!
+
+Acknowledgments
+Sincerely appreciate the Microverse Team for igniting this journey
+etc
+
+📝 License
+
+ License
+This project is MIT licensed.
```
Comment (omarsalem7) [inline README.md]: - [ ] As stated by the previous reviewer, you should use a microverse template and the same markdown syntax, you need to copy syntax also not the text, so in [this link](https://raw.githubusercontent.com/microverseinc/readme-template/master/README.md) you can copy text with syntax and customize it for your repo to make it a professional one.

**Review by omarsalem7 — CHANGES_REQUESTED** [review]
Hi @canon20paul , 

Great work on making the changes requested by a previous reviewer 👏 
You've done well implementing some of the requested changes, but there are still some which aren't addressed yet.

Almost near

 ## Highlights
✔️ All linter checks pass
✔️ Descriptive pull request

 ## Required Changes ♻️
_Check the comments under the review._

## Optional suggestions
_Every comment with the [OPTIONAL] prefix won't stop the approval of this PR. However, I strongly recommend you to take them into account as they can make your code better. Some of them were simply missed by the previous reviewer and addressing them will really improve your application._


Cheers and Happy coding!👏👏👏

Feel free to leave any questions or comments in the PR thread if something is not 100% clear.
Please, remember to tag me in your question so I can receive the notification.

**_Please, do not open a new Pull Request for re-reviews. You should use the same Pull Request submitted for the first review, either valid or invalid unless it is requested otherwise._**


------
_As described in the [Code reviews limits policy](https://microverse.zendesk.com/hc/en-us/articles/1500004088561)  you have a limited number of reviews per project (check the exact number in your Dashboard). If you think that the code review was not fair, you can request a second opinion using [this form](https://airtable.com/shrQAqnBwek5a0O0s)._

**Review by 1GORDON — APPROVED** [review]
### **Hi @canon20paul ,** 

### **Project Approved ✅**

Approved

## Optional Changes:
Please try to rename your Pull request title to something more descriptive like. 
- Milestone One: Hello World Project

Your project is complete!  There is nothing else to say other than... it's time to merge it 🚀
Congratulations! 🎉 

Cheers and Happy coding!👏👏👏

Feel free to leave any questions or comments in the PR thread if something is not 100% clear.

------
_As described in the [Code reviews limits policy](https://microverse.zendesk.com/hc/en-us/articles/1500004088561)  you have a limited number of reviews per project (check the exact number in your Dashboard). If you think that the code review was not fair, you can request a second opinion using [this form](https://airtable.com/shrQAqnBwek5a0O0s)._

**Pull request merged**